src/open_inwoner/core/views.py

```diff
@@ -2,10 +2,12 @@
 
 from django.shortcuts import render
 from django.urls import reverse
+from django.utils.translation import gettext as _
 
 from cms.models import Title
 
 from open_inwoner.accounts.models import User
+from open_inwoner.cms.profile.cms_appconfig import ProfileConfig
```

Comment:
How is this handled in the sitemap template?
When running the PR i get these errors:
"MultipleObjectsReturned at /sitemap/
get() returned more than one ProfileConfig -- it returned 2!"

Reply:
I don't think we have a use case for multiple instances of `ProfileConfig`, so I suspect there's some issue with your data. However, I amended the code to handle this situation (can't know for sure if someone did not create multiple profile configurations in prod somehow).